docs/specs/index.md

```diff
@@ -1,22 +1,37 @@
-# Dusty Specs Overview
-There are 4 main types of objects in Dusty.<br/>
-1. [Bundles](./bundle-specs.md) - highest level objects in Dusty.  They can be activated or deactivated.  Bundles are organized in logical groups, like the website or api. `dusty up` will run the set of activated bundles.<br/>
-2. [Apps](./app-specs.md) - objects that should run in their own container and have code from the user.<br/>
-3. [Services](./service-specs.md) - objects that should run in their own container but uses external code (mongo or redis instance).<br/>
-4. [Libs](./lib-specs.md) - objects that have code from the user, but do not run in their own container. These will be included in app containers for app code to use.
+# Specs Overview
 
-There is a 5th subtype that is important enough to get its own spec page.<br/>
-5. [Tests](./test-specs.md) - objects found on both the lib and apps objects. They define how to run tests on a user's app and lib code.
+Dusty uses YAML specifications for configuration of the applications, services, and
+libraries used to create your stack.
+
+An additional entrypoint layer, called a Bundle, is used to provide toggleable entrypoints
+into your stack's dependency graph. Users can decide which Bundles they want to run,
+and Dusty runs the applications, services, and libraries defined by those Bundles.
+
+1. [Bundles](./bundle-specs.md): User-toggleable logical groups of applications.
```

Comment:
User-toggleable logical groups is a mouth full.   Logical groups of applications users can toggle?